src/App.tsx

```diff
@@ -0,0 +1,35 @@
+import { Navbar } from "./components";
+import { BrowserRouter, Routes, Route } from "react-router-dom";
+import Home from "./pages/Home";
+import About from "./pages/About";
+import background from "./assets/background.png";
+import CountdownTimer from "./components/CountDownTimer";
+
+export const App = () => {
+  return (
+    <BrowserRouter>
+      <div className="min-width: 1920px; margin: 0 auto; max-height: 1080px;">
```

Comment:
min-width 1920px? How will this look on mobile?

Reply:
Going to fix this issue in another PR :)